handlers/anteHandler.go

```diff
@@ -109,7 +109,7 @@ func validateInput(ctx sdk.Context, input plasma.Input, signer common.Address, u
 	if inputUTXO.Spent {
 		return nil, msgs.ErrInvalidTransaction(DefaultCodespace, "input, %v, already spent", input.Position).Result()
 	}
-	if client.HasTxBeenExited(input.Position) {
```

Comment:
I do not think this will function correctly. TMBlockHeight != plasmaBlocks submitted to rootchain. For example, the tmBlock can be at 150,000 where as the last submitted block to the rootchain can be 24. You might need to add a GetCurrentPlasmaBlock function to plasmaStore. I think `tmBlock` naming should also be changed to `plasmaBlock` to avoid confusion

Reply:
Yup forgot about that. And yes, we should use `plasmaBlock`